site2/docs/helm-deploy.md

```diff
@@ -131,47 +134,47 @@ The Pulsar Helm chart is designed to enable controlled upgrades. So it can confi
 images:
   zookeeper:
     repository: apachepulsar/pulsar-all
-    tag: 2.5.0
+    tag: @pulsar:version@
     pullPolicy: IfNotPresent
   bookie:
     repository: apachepulsar/pulsar-all
-    tag: 2.5.0
+    tag: @pulsar:version@
     pullPolicy: IfNotPresent
   autorecovery:
     repository: apachepulsar/pulsar-all
-    tag: 2.5.0
+    tag: @pulsar:version@
     pullPolicy: IfNotPresent
   broker:
     repository: apachepulsar/pulsar-all
-    tag: 2.5.0
+    tag: @pulsar:version@
     pullPolicy: IfNotPresent
   proxy:
     repository: apachepulsar/pulsar-all
-    tag: 2.5.0
+    tag: @pulsar:version@
     pullPolicy: IfNotPresent
   functions:
     repository: apachepulsar/pulsar-all
-    tag: 2.5.0
+    tag: @pulsar:version@
   prometheus:
```

Comment:
@michaeljmarshall I believe you have updated many of these tags and repositories.

Reply:
@michaeljmarshall did you see any updates to be addressed in these docs following the changes you made to the helm chart?

Reply:
ping @michaeljmarshall 

Since Pulsar Helm Charts 3.0.0 has been released, could you review this part of documents?

Reply:
This section looks good to me.